### PR TITLE
Currency: Added build-script and build-info.json for tomcat

### PR DESCRIPTION
**Pull request by Chandranana**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: t/tomcat/build_info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "tomcat",
+    "github_url": "https://github.com/apache/tomcat.git",
+    "version": "11.0.0-M3",
+    "default_branch": "main",
+    "package_dir": "t/tomcat/",
+    "build_script": "tomcat_ubi_8.5.sh",
+    "validate_build_script": true,
+    "use_non_root_user": true,
```
Comment (nikhil-kalbande) [inline t/tomcat/build_info.json:9]: please make this "use_non_root_user": true, ---> "use_non_root_user": false.

Here `use_non_root_user: true -- normal user without sudo permission`
`use_non_root_user:false -- normal user with sudo permission`

File: t/tomcat/tomcat_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# -----------------------------------------------------------------------------
+#
+# Package	: tomcat
+# Version	: v11.0.0-M3
+# Source repo	: https://github.com/apache/tomcat.git
+# Tested on	: ubi 8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Haritha Patchari <haritha.patchari@ibm.com>
```
Comment (nikhil-kalbande) [inline t/tomcat/tomcat_ubi_8.5.sh:11]: add your name as maintainer here

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: t/tomcat/build_info.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "package_name" : "tomcat",
```
Comment (nikhil-kalbande) [inline t/tomcat/build_info.json:2]: we like to add apache packages like `apache-tomcat`. So no change need in build script. But change package_name here.

Also not build script and build_info.json should added in folder a/apche-tomcat

I see we already have folder also - https://github.com/ppc64le/build-scripts/tree/master/a/apache-tomcat.
Please make necessary changes

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: t/tomcat/tomcat_ubi_8.5.sh (hunk 2 above)
Comment (Chandranana) [reply t/tomcat/tomcat_ubi_8.5.sh]: Done

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**